apps/src/code-studio/pd/professional_learning_landing/landingPage.jsx

```diff
@@ -1,118 +0,0 @@
-/**
- * Teacher Landing Page
- */
-
-import React, {PropTypes} from 'react';
-import {Button} from 'react-bootstrap';
-import ProfessionalLearningCourseProgress from './professionalLearningCourseProgress';
-import {TwoPartBanner} from './twoPartBanner';
-import {EnrolledWorkshops} from './enrolledWorkshops';
-import _ from 'lodash';
-
-const LandingPage = React.createClass({
-  propTypes: {
-    lastWorkshopSurveyUrl: PropTypes.string,
-    lastWorkshopSurveyCourse: PropTypes.string,
-    professionalLearningCourseData: PropTypes.array
-  },
-
-  renderHeaderImage() {
-    return (
-      <div
-        style={{
-          width: '100%',
-          height: '300px',
-          background: `url(/blockly/media/BannerKids.png) no-repeat`,
-          backgroundSize: 'cover',
-          display: 'flex',
-          alignItems: 'center',
-          justifyContent: 'center'
-        }}
-      >
-        <div
-          style={{
-            backgroundColor: 'rgba(0, 0, 0, .5)',
-            alignSelf: 'flex-end',
-            width: '100%',
-            textAlign: 'center',
-            padding: '30px',
-            fontSize: '40px',
-            color: 'white'
-          }}
-        >
-          My Professional Learning
-        </div>
-      </div>
-    );
-  },
-
-  renderWorkshopSurveyInterior() {
-    if (this.props.lastWorkshopSurveyCourse === 'CS Fundamentals') {
-      return (
-        <div>
-          <h3>
-            Submit feedback and order free course kit
-          </h3>
-          <p>
-            Thank you for taking a CS Fundamentals workshop! Please complete this survey about your experience and you
-            will be able to order supplies for your classroom.
-          </p>
-          <Button bsStyle="primary" onClick={() => {window.open(this.props.lastWorkshopSurveyUrl, '_blank');}}>
```

Comment:
Any reason this `onClick` callback was removed? Now it the start survey button appears to be totally broken, redirecting to `/` instead of the survey url.

Reply:
I'm owning this fix, will make sure it goes out tomorrow.